docs/pages/connect-your-client/tsh.mdx

```diff
@@ -842,5 +842,59 @@ DEBU [TSH]       Self re-exec command: tsh [status --format=json]. tsh/aliases.g
 ...
 ```
 
+# Interactive Recording
+
+Within the Teleport cluster the `ssh` and `Kubernetes` recorded sessions are available for listing and replay for authorized users via `tsh`.
+The same listings including Desktops are available in the Web Console under the Audit Log in the Activity section.
```

Comment:
Recordings are listed under "Session Recordings" - not under audit log, right?

Reply:
```suggestion
The same listings, including Desktop Access sessions, are available in the Web Console under the Audit Log in the Activity section.
```